D:\New\lonelybot\python\lonelybot_py\target\debug\build\thiserror-f340d441dc92b089\out